nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ * (C) Copyright 2020 Nuxeo SA (http://nuxeo.com/).
+ * This is unpublished proprietary source code of Nuxeo SA. All rights reserved.
+ * Notice of copyright on this source code does not indicate publication. *
+ *
+ *
+ * Contributors:
+ *     Nuxeo
+ */
+
+package org.nuxeo.ai;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+import javax.ws.rs.core.Response;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ai.enrichment.EnrichmentTestFeature;
+import org.nuxeo.ecm.platform.test.PlatformFeature;
+import org.nuxeo.ecm.restapi.test.BaseTest;
+import org.nuxeo.ecm.restapi.test.RestServerFeature;
+import org.nuxeo.elasticsearch.test.RepositoryElasticSearchFeature;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+@RunWith(FeaturesRunner.class)
+@Features({ RestServerFeature.class, EnrichmentTestFeature.class, PlatformFeature.class,
+        RepositoryElasticSearchFeature.class })
+@Deploy({ "org.nuxeo.ai.ai-core", "org.nuxeo.ai.ai-config", "org.nuxeo.ai.ai-model" })
+@Deploy({ "org.nuxeo.ai.ai-config.test:test-es-contrib.xml", "org.nuxeo.elasticsearch.http.readonly" })
+public class TestAISearchREST extends BaseTest {
+
+    protected String payload = "{\n" + "    \"query\": {\n" + "        \"bool\": {\n" + "            \"must\": [\n"
```

Comment:
rather put this in a json resource file

Reply:
I think im gonna do a small one without so much stuff